docs/source/topics/processes/usage.rst

```diff
@@ -350,6 +350,28 @@ The function will submit the calculation to the daemon and immediately return co
 .. warning::
     For a process to be submittable, the class or function needs to be importable in the daemon environment by a) giving it an :ref:`associated entry point<how-to:plugin-codes:entry-points>` or b) :ref:`including its module path<how-to:faq:process-not-importable-daemon>` in the ``PYTHONPATH`` that the daemon workers will have.
 
+.. tip::
+    Use ``wait=True`` when calling ``submit`` to wait for the process to complete before returning the node.
+    This can be useful for tutorials and demos in interactive notebooks where the user should not continue before the process is done.
+    One could of course also use ``run`` (see below), but then the process would be lost if the interpreter gets accidentally shut down.
+    By using ``submit``, the process is run by the daemon which takes care of saving checkpoints so it can always be restarted in case of problems.
+    If you need to launch multiple processes in parallel and want to wait for all of them to be finished, simply use ``submit`` with the default ``wait=True`` and collect the returned nodes in a list.
+    You can then pass them to :func:`aiida.engine.launch.await_processes` which will return once all processes have terminated::
+
+        .. code:: python
+            from aiida.engine import submit, await_processes
+
+            nodes = []
+
+            for i in range(5):
+                node = submit(...)
+                nodes.append(node)
+
+            await_processes(nodes, sleep_interval=10)
+
+    The ``await_processes`` function will loop every ``sleep_interval`` seconds and check whether all processes (represented by the ``ProcessNode``s in the ``nodes`` list) have terminated.
```

Comment:
I think the extra `s` after the double backticks is causing trouble here.

![Screenshot 2023-10-26 at 22 39 59](https://github.com/aiidateam/aiida-core/assets/25607960/d90a21a3-64a4-47dd-b69c-4f650bfb4f1c)

Reply:
ah yeah I forgot. Restructured text cannot parse this properly. Absolute garbage...